integration-test/https-server/server.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"crypto/tls"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	certFile = "cabundle/cert.pem"
```

Comment:
Just a minor suggestion:
```suggestion
	certFile = path.Join("cabundle", "cert.pem")
```
If you do add this `path` will need to be added to the imports + `keyFile` should be updated as well.